boards/FSM/Inc/Io/Io_MSP3002K5P3N1.h

```diff
@@ -1,15 +1,27 @@
 #pragma once
 
+#include <stdbool.h>
+
 /**
  * Check if the pressure from MSP-300-2K5-P-3-N-1 exceeds the brake pressure
  * threshold. If the pressure exceeds threshold, the brake is said to be
- * actuated.
+ * actuated
  * @return True if the brake is actuated, false if the brake is not actuated
  */
 bool Io_MSP3002K5P3N1_IsBrakeActuated(void);
```

Comment:
As a general thought for these functions, is there anything to `MSP3002K5P3N1` sensor that is actually related to braking if we're not looking at it in the context of our car? The point I'm trying to make here is that this should solely be an abstraction around the `MSP3002K5P3N1`. An abstraction around the brakes might have an instance of this as part of it. 

Hope that's clear. Not super critical for this PR, but just a thought.

Reply:
I'm still a bit confused. In this case are you suggesting that functions like `IsBrakeActuated` and `IsOpenOrShortCircuit` should fall under the abstraction around the brakes instead of the `MSP3002K5P3N1` sensor?

Reply:
I agree with Gareth. I think IsBrakeActuated should go elsewhere while IsOpenOrShortCircuit can stay since it's directly related to the sensor itself

Reply:
Not sure what the exact kind of abstraction @garethellis0 had in mind

Reply:
I was just making a comment that generally we should only put stuff under `Io_MSP3002K5P3N1` that is _specific_ to the `MSP3002K5P3N1`. So if you think of the `MSP3002K5P3N1`, does it have the concept of "isBrakeActuated" or `isOpenOrShortCircuit"? Or are those things that we are associating with it because of our specific use-case of it? If they are not specific to this sensor, then we should have an `App_Brake` that wraps around it.

Sorry if this is a bit confusing, it's a little hard for me to convey in text. Let me know if you want to hope in a call to clarify it quickly!

Reply:
Oh ok! thx for clearing this up